src/ResourceMover/README.md

```diff
@@ -132,7 +132,7 @@ directive:
        suppress-format: true
   - no-inline:
     - ResourceSettings
-  - from: source-file-csharp
+  - from: ResourceMover.cs
     where: $
-    transform: $ = $.replace(/error = true;/g, '//error = true;');
+    transform: $ = $.replace(/throw new Microsoft.Azure.PowerShell.Cmdlets.ResourceMover.Runtime.UndeclaredResponseException\(_response\);/g,"await onDefault\(_response,_response.Content.ReadAsStringAsync\(\).ContinueWith\( body => Microsoft.Azure.PowerShell.Cmdlets.ResourceMover.Models.Api20230801.CloudError.FromJson\(Microsoft.Azure.PowerShell.Cmdlets.ResourceMover.Runtime.Json.JsonNode.Parse\(body.Result\)\) \)\);");
```

Comment:
There are OperationStatusError defined in swagger, please be aware that by using directive at line 137, response will be deserialized into CloudError instead of OperationStatusError.

please double check and confirm when status code is 200 and provisioning state failed, the response could match the schema of clouderror.

Reply:
OperationStatusError is compatible when parsing into CloudError for error reporting.